pulsar-broker-common/src/main/java/org/apache/pulsar/broker/authentication/AuthenticationState.java

```diff
@@ -47,7 +49,14 @@
     AuthenticationDataSource getAuthDataSource();
 
     /**
-     * Whether the authentication is completed or not
+     * Whether the authentication is completed or not.
      */
     boolean isComplete();
+
+    /**
+     * Get AuthenticationState ID
+     */
+    default long getStateId() {
+        return 0;
```

Comment:
should you return -1L? since 0 is a valid  state id.

Reply:
0 will not return.